src/USER-CONP/ewald_conp.h

```diff
@@ -85,6 +85,9 @@ class EwaldConp : public KSpace {
   void wirecorr_groups(int, int, int);
   void allocate_groups();
   void deallocate_groups();
+ 
+ private:
+  int eikr_step;
```

Comment:
Was macht das genau?

Reply:
compute_vector() und so müssen eik_dot_r() aufrufen, aber nur wenn das in dem Zeitschritt noch nicht gemacht wurde, weil es z.B.  compute_vector_corr() aufgerufen haben könnte.

Reply:
eikr_step speichert also den Zeitschritt an dem das letzte Mal eik_dot_r gemacht wurde?

Reply:
Ja, die compute()-Methode benutzt das aber nicht, weil die das ja ein zweites Mal ausführen muss.